rclcpp/src/rclcpp/node_interfaces/node_services.cpp

```diff
@@ -41,14 +41,12 @@ NodeServices::add_service(
   }
 
   // Notify the executor that a new service was created using the parent Node.
-  {
-    auto notify_guard_condition_lock = node_base_->acquire_notify_guard_condition_lock();
-    if (rcl_trigger_guard_condition(node_base_->get_notify_guard_condition()) != RCL_RET_OK) {
-      throw std::runtime_error(
-              std::string("Failed to notify wait set on service creation: ") +
-              rmw_get_error_string().str
-      );
-    }
+  auto & node_gc = node_base_->get_notify_guard_condition();
+  try {
+    node_gc.trigger();
+  } catch (const rclcpp::exceptions::RCLError & ex) {
+    throw std::runtime_error(
+            std::string("Failed to notify wait set on service creation: ") + ex.what());
```

Comment:
I know it was like this before, but the predominate style elsewhere is lower case:

```suggestion
            std::string("failed to notify wait set on service creation: ") + ex.what());
```

Reply:
Done in f99baec